ydb/mvp/core/core_ydb.cpp

```diff
@@ -109,20 +104,24 @@ std::unique_ptr<NYdb::NTopic::TTopicClient> TYdbLocation::GetTopicClientPtr(TStr
     return std::make_unique<NYdb::NTopic::TTopicClient>(GetDriver(endpoint, scheme), settings);
 }
 
-NYdb::NTable::TTableClient TYdbLocation::GetTableClient(const TRequest& request, const NYdb::NTable::TClientSettings& defaultClientSettings) const {
+NYdb::NTable::TTableClient TYdbLocation::GetTableClient(const TRequest& request, const TYdbLocation& location) const {
```

Comment:
it's better to keep the settings in the parameter.

and why do you need location parameter in a method of location class?

Reply:
kept the settings parameter